bin/dump-dev-program-config

```diff
@@ -7,17 +7,18 @@ source bin/lib.sh
 docker::set_network_name_dev
 
 docker run --rm -it \
+  -v "$(pwd):/project" \
   -e "PGPASSWORD=example" \
   --network "${DOCKER_NETWORK_NAME}" \
   postgres:12.5 \
-  /usr/bin/pg_dump \
-  -w \
-  -Fc \
-  -h db \
-  -U postgres \
-  -t programs \
-  -t questions \
-  -t versions \
-  -t versions_programs \
-  -t versions_questions \
-  postgres > dev_programs.dump
+  /bin/bash -c "pg_dump \
```

Comment:
Why was this changed from /usr/bin? And why does the other file still use usr/bin ?

Reply:
tbh mostly habit and tradition, at least for bash.

system binaries are mostly in `/bin` whereas user binaries often go in `/usr/bin`. it's common to invoke the system version of bash and afaik on most linux distros `/bin/bash` and `/usr/bin/bash` are actually the same inode.

in the case of `pg_dump`, it's userland program rather than a system utility, so it's unlikely to ever be in `/bin`